components/common/src/lib.rs

```diff
@@ -37,8 +37,15 @@ pub mod cli_defaults;
 pub mod command;
 pub mod error;
 pub mod locked_env_var;
+pub mod output;
 pub mod package_graph;
 pub mod templating;
 pub mod types;
 pub mod ui;
 pub mod util;
+
+lazy_static::lazy_static! {
+    pub static ref PROGRAM_NAME: String = {
+        std::env::current_exe().unwrap().file_stem().and_then(|p| p.to_str()).unwrap().to_string()
```

Comment:
[`current_exe` is not infallible](http://doc.rust-lang.org/1.33.0/std/env/fn.current_exe.html#errors). Are we sure we want to abort the program if it fails? Maybe it's better to log an `error!` and return some default like `"hab-?"`